src/trace_processor/trace_processor_shell.cc

```diff
@@ -1441,26 +1440,26 @@ base::Status IncludeSqlPackage(TraceProcessor* trace_processor,
   std::vector<std::string> paths;
   RETURN_IF_ERROR(base::ListFilesRecursive(root, paths));
   sql_modules::NameToPackage modules;
-  for (const auto& path : paths) {
-    if (base::GetFileExtension(path) != ".sql") {
+  for (const auto& file_path : paths) {
+    if (base::GetFileExtension(file_path) != ".sql") {
       continue;
     }
-
-    std::string path_no_extension = path.substr(0, path.rfind('.'));
-    if (path_no_extension.find('.') != std::string_view::npos) {
+    std::string file_name = BaseName(file_path);
```

Comment:
This is *not* waht this function used to do. This only returns the filename. What it used to do before was the path relative to the root.

This is not the same.